build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
Of course it requires a commit - It doesn't require a modification to `build.gradle` is what I was saying. :)

Do you want me to add the custom `wrapper` task back in?